packages/browser/src/client.ts

```diff
@@ -57,6 +57,15 @@ export class BrowserClient extends BaseClient<BrowserBackend, BrowserOptions> {
     });
   }
 
+  /**
+   * @inheritDoc
+   */
+  public captureSession(session: Session): void {
+    // Make sure session duration is not sent
```

Comment:
I thought of something less drastic than this, to only omit durations for the automatically instrumented session, not all calls to `captureSession`, but I guess this could work too.

The comment should explain more of the "why", the "what" is clear from the code itself:

```suggestion
    // The session duration for browser sessions does not track a meaningful
    // concept that can be used as a metric.
    // Automatically captured sessions are akin to page views, and thus we
    // discard their duration.
```

Feel free to reword.